src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -14,6 +14,15 @@ internal sealed class CoreEventSource : EventSource
 
         private readonly ApplicationNameProvider nameProvider = new ApplicationNameProvider();
 
+        public bool IsVerboseEnabled
```

Comment:
We never had a need for them. IsVeroseEnabled is used when additional tracing has performance implications. 

---
In reply to: [64499033](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/232#discussion_r64499033) [](ancestors = 64499033)